<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PasswordWindow</class>
 <widget class="QMainWindow" name="PasswordWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>524</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-10</y>
      <width>551</width>
      <height>571</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#widget{
	border-image: url(:/IMG_2027.JPG);}
</string>
    </property>
    <widget class="QPushButton" name="retrievePasswordPushButton">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>420</y>
       <width>241</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 16pt &quot;Arial&quot;;
color: rgb(0, 0, 0);
background-color: rgb(255, 170, 127);</string>
     </property>
     <property name="text">
      <string>找回密码</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="IDLineEdit">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>280</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(145, 145, 145,120);
	border-radius:3px;
	color:rgb(0,0,0);
font: 12pt &quot;黑体&quot;;
border:1px solid #e1e0e0;
border-color: rgb(255, 255, 255);

</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="UsernameLineEdit">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>130</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(145, 145, 145,120);
	border-radius:3px;
	color:rgb(0,0,0);
font: 12pt &quot;黑体&quot;;
border:1px solid #e1e0e0;
border-color: rgb(255, 255, 255);


</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="TelphoneLineEdit">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>200</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(145, 145, 145,120);
	border-radius:3px;
	color:rgb(0,0,0);
font: 12pt &quot;黑体&quot;;
border:1px solid #e1e0e0;
border-color: rgb(255, 255, 255);

</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>280</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">	bcakground-color: rgb(255,132,139,0);
font: 12pt &quot;04b_21&quot;;
	border-radius:3px;
	color:rgb(255,255,255);</string>
     </property>
     <property name="text">
      <string>身份证号</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>210</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">	bcakground-color: rgb(255,132,139,0);
font: 12pt &quot;04b_21&quot;;
	border-radius:3px;
	color:rgb(255,255,255);</string>
     </property>
     <property name="text">
      <string>手机号</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>130</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">	bcakground-color: rgb(255,132,139,0);
font: 12pt &quot;04b_21&quot;;
	border-radius:3px;
	color:rgb(255,255,255);</string>
     </property>
     <property name="text">
      <string>用户名</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="doctorRadioButton">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>360</y>
       <width>115</width>
       <height>19</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>医生</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="userRadioButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>360</y>
       <width>115</width>
       <height>19</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>用户</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>524</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
